visualization/index.html

```diff
@@ -113,16 +113,16 @@ <h1>
 							</select>
 						</div>
 						<div class="selector">
-							<p class="key noselect">Ordenar por</p>
+							<p class="key noselect">Sort by</p>
 							<select name="sort" id="sort" class="select value noselect">
-								<option name="article" value="1" aria-label="nombre del artículo">nombre del artículo</option>
-								<option name="publication" value="2" aria-label="fecha de publicación">fecha de publicación</option>
-								<option name="size" value="3" aria-label="tamaño del artículo">tamaño del artículo</option>
-								<option name="discussion" value="4" aria-label="tamaño de la página de discusión">tamaño de la página de discusión</option>
-								<option name="incipit" value="5" aria-label="tamaño de la introducción">tamaño de la introducción</option>
-								<option name="issue" value="6" aria-label="número de avisos">número de avisos</option>
-								<option name="images" value="7" aria-label="número de imágenes">número de imágenes</option>
-								<option name="reference" value="8" aria-label="referencias">referencias</option>
+								<option name="article" value="1" aria-label="article name">article name</option>
+								<option name="publication" value="2" aria-label="publication date">publication date</option>
+								<option name="size" value="3" aria-label="article size">article size</option>
+								<option name="discussion" value="4" aria-label="talk page size">talk page size</option>
+								<option name="incipit" value="5" aria-label="introduction size">introduction size</option>
+								<option name="issue" value="6" aria-label="number of alerts">number of alerts</option>
+								<option name="images" value="7" aria-label="number of images">number of images</option>
+								<option name="reference" value="8" aria-label="references">references</option>
```

Comment:
this should be "number of references" to maintain the same style as in the other options